android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -37,7 +38,7 @@ class TranslationsMiddleware(
 ) : Middleware<BrowserState, BrowserAction> {
     private val logger = Logger("TranslationsMiddleware")
 
-    @Suppress("LongMethod")
+    @Suppress("LongMethod","CyclomaticComplexMethod")
```

Comment:
Something between updates triggered `CyclomaticComplexMethod` lint. Think it might be the `?:` operator [pushed it over the limit](https://detekt.dev/docs/rules/complexity/#cyclomaticcomplexmethod)? (Not much has changed besides that, unless a global rule changed on the rebase.)

Reply:
So weird that it could complain about the `invoke` method when the `?:` operators are all appearing in smaller methods.

Reply:
Showed up in #5726 on rebase. Change there was just adjusting around how `Init` occurred. (Patch was written before the refactor to how init worked in translations middleware via #5810.) 🧐 Weird